src/llmcompressor/transformers/finetune/README.md

```diff
@@ -61,13 +60,13 @@ train(
     model=model,
     distill_teacher=teacher_model,
     dataset=dataset_name,
-    output_dir=output_dir,
```

Comment:
It occurs to me that removing saving within the oneshot function also removes user's ability to pass a string name into the model argument, since there is no way to save the compressed model if only a string is passed.

This is critical functionality we want to support, so we should definitely continue to allow users to save via oneshot, and we should also guarantee check arguments to guarantee that if a string is passed for the model argument, then output_dir is not empty

Reply:
ok

Reply:
add back output dir to the example